<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TorchQT</class>
 <widget class="QDialog" name="TorchQT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1">
     <item>
      <widget class="QGroupBox" name="input">
       <property name="title">
        <string>Input</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,1,1,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="input_img">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="input_btn">
            <property name="text">
             <string>Image Upload</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Model</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="1">
              <widget class="QComboBox" name="mode_box">
               <item>
                <property name="text">
                 <string>cam</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>grad cam</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>guided backprop</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>vanilla grad</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>smooth grad</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="cls_box">
               <property name="minimum">
                <number>-1</number>
               </property>
               <property name="maximum">
                <number>999</number>
               </property>
               <property name="value">
                <number>-1</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="mode_label">
               <property name="text">
                <string>Mode</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="model_label">
               <property name="text">
                <string>Model</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="model_box">
               <item>
                <property name="text">
                 <string>resnet18</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="cls_btn">
               <property name="text">
                <string>Class Number</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start_btn">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>SHOW</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QLabel" name="main_label">
          <property name="text">
           <string>Upload the image</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QListWidget" name="layers_widget"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>GitHub : https://github.com/jjeamin</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
